packages/bpk-component-barchart/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bpk-component-barchart",
-  "version": "2.1.27",
+  "version": "2.1.28",
```

Comment:
Again the person who is doing the release to npm will take care of this (actually lerna takes care of it for them!). Please revert.

Reply:
It's a usability linter error: 
> [eslint] onMouseOver must be accompanied by onFocus for accessibility. ([jsx-a11y/mouse-events-have-key-events](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/mouse-events-have-key-events.md))